projects/plugins/jetpack/extensions/blocks/subscriber-login/block.json

```diff
@@ -42,6 +42,10 @@
 		"logOutLabel": {
 			"type": "string"
 		},
+		"showLinkToManageSubscriptions": {
```

Comment:
Just a nitpick but I would rename it to `showManageSubscriptionsLink`. It's a bit shorter.